src/CMakeLists.txt

```diff
@@ -0,0 +1,50 @@
+set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+set(LINK_WITH_SHARED_OSI OFF CACHE BOOL "Link FMU with shared OSI library instead of statically linking")
+set(PUBLIC_LOGGING OFF CACHE BOOL "Enable logging via FMI logger")
+set(PRIVATE_LOGGING OFF CACHE BOOL "Enable private logging to file")
+
+string(TIMESTAMP FMUTIMESTAMP UTC)
+string(MD5 FMUGUID modelDescription.in.xml)
+configure_file(modelDescription.in.xml modelDescription.xml @ONLY)
+
+find_package(Protobuf 2.6.1 REQUIRED)
+add_library(COSMPCameraSensor SHARED COSMPCameraSensor.cpp COSMPCameraSensor.h)
```

Comment:
In this line it has to say "OSMPCameraSensor.cpp OSMPCameraSensor.h" without the "C"s at the beginning.